app/src/main/java/com/example/learn09082021v001/NameScoreOOP.kt

```diff
@@ -25,11 +26,12 @@ class NameScoreOOP (
     }
 
 }
-class datasaveandload
-{
+
+class datasaveandload {
```

Comment:
don't naming a class like this. Check https://developer.android.com/kotlin/style-guide#naming